src/components/shared/Wizard/types.ts

```diff
@@ -1,8 +1,6 @@
 import { AppContextValues } from '~context/AppContext';
-import { Values } from '~types';
 
-type WizardValues<T> = Record<string, Values<T>>;
-export type StepValues<T> = Partial<WizardValues<T>>;
```

Comment:
@willm30 For some reason, this was interfering with introducing any fields that are not of primitive type (e.g. the `token` field). It would cause the types of all the other fields to become `undefined`. Removing it didn't throw any error and as far as I can tell the types still work. Would appreciate if you could check too

Reply:
LGTM